actions/ostree_deploy_action.go

```diff
@@ -121,6 +121,12 @@ func (ot *OstreeDeployAction) Run(context *debos.DebosContext) error {
 	repoPath := "file://" + path.Join(context.Artifactdir, ot.Repository)
 
 	sysroot := ostree.NewSysroot(context.Rootdir)
+
+	/* libostree keeps some information, like repo lock file descriptor, in
+	 * thread specific variables. As GC can be run from another thread, it
+	 * may not been able to access this, preventing to free them correctly.
+	 * To prevent this, explicitly dereference libostree objects. */
+	defer sysroot.Unref()
```

Comment:
Calling `Unref()` doesn't seem to be safe, as there's a finalizer set to call `Unref()` as well, see https://github.com/sjoerdsimons/ostree-go/blob/master/pkg/otbuiltin/sysroot.go#L36